src/amo/components/CategoryHeader/index.js

```diff
@@ -1,95 +1,55 @@
+/* @flow */
 import makeClassName from 'classnames';
 import * as React from 'react';
-import PropTypes from 'prop-types';
-import { compose } from 'redux';
 
 import CategoryIcon from 'amo/components/CategoryIcon';
-import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
-import translate from 'core/i18n/translate';
 import { getCategoryColor } from 'core/utils';
 import Card from 'ui/components/Card';
 import LoadingText from 'ui/components/LoadingText';
+import type { CategoryType } from 'amo/types/categories';
 
 import './styles.scss';
 
-export class CategoryHeaderBase extends React.Component {
-  static propTypes = {
-    category: PropTypes.object,
-    i18n: PropTypes.object.isRequired,
-  };
+type Props = {|
+  category: CategoryType | null,
+|};
 
+export class CategoryHeaderBase extends React.PureComponent<Props> {
   render() {
-    const { category, i18n } = this.props;
+    const { category } = this.props;
 
     const addonType = category && category.type ? category.type : null;
 
-    let color = 0;
     let icon = null;
+    let className = null;
 
-    if (category) {
-      color = getCategoryColor(category);
+    if (category && addonType) {
       icon = category.slug === 'other' ? `other-${addonType}` : category.slug;
-    }
-
-    // This is here until
-    // https://github.com/mozilla/addons-server/issues/5728
-    // as well as
-    // https://github.com/mozilla/addons-server/issues/5729 are fixed.
-    // Also, if we don't get a description for a category (maybe it's a new one
-    // or lacks a description) we'll show this generic message.
-    if (category && !category.description) {
-      category.description = i18n.gettext(
-        'Browse all add-ons in this category.',
-      );
-
-      if (category.type === ADDON_TYPE_EXTENSION) {
-        category.description = i18n.gettext(
-          'Browse all extensions in this category.',
-        );
-      }
-
-      if (category.type === ADDON_TYPE_THEME) {
-        category.description = i18n.gettext(
-          'Browse all themes in this category.',
-        );
-      }
+      className = `CategoryHeader--type-${addonType}`;
```

Comment:
as far as I can tell, there are no styles associated with this classname. Just checking..do we still need this?

Reply:
right let's remove it, thanks!